test/operator.jl

```diff
@@ -547,9 +547,12 @@ function operators_test(ModelType::Type{<:JuMP.AbstractModel}, VariableRefType::
             @test_throws ErrorException A./y
             @test_throws ErrorException B./y
 
-            @test JuMP.isequal_canonical((2 .* x) ./ 3, Matrix((2 .* y) ./ 3))
-            @test JuMP.isequal_canonical(2 .* (x ./ 3), Matrix(2 * (y ./ 3)))
-            @test JuMP.isequal_canonical((x[1,1],) .* A, Matrix((x[1,1],) .* B))
+            z = JuMP.densify_with_jump_eltype((2 .* y) ./ 3)
```

Comment:
Using an internal JuMP function to construct the expected answer seems unnecessary and makes it harder to verify that the test is correct.

Reply:
Yeah - before this PR, these tests' reliance on internal constructors was obscured by the fact that the constructors were pirated overloads of `Matrix`. Now that it's easily visible where this is happening, I've added a TODO to refactor the tests